src/Components/App/App.test.js

```diff
@@ -59,24 +65,63 @@ describe('App', () => {
     expect(singlePokemon).toBeInTheDocument();
   });
 
-  it('should start with a winning streak of 0, and add 1 if the correct answer is selected', async () => {
-    const { debug, getByRole, getByText, getAllByRole, getByPlaceholderText } = render(
+  it('should have 4 pokemon choices buttons when navigating to /game', async () => {
+    const { getByText, getAllByRole } = render(
       <MemoryRouter>
         <App />
       </MemoryRouter>
     );
     const gameNav = getByText('Game')
     fireEvent.click(gameNav);
+    const pokemonChoices = await waitFor(() => getAllByRole('button'));
+    expect(pokemonChoices.length).toBe(4);
+  });
+
+  it('should start with a winning streak of 0, add 1 if the correct answer is selected, and reset to 0 if wrong answer selected', async () => {
+    const { getByRole, getByText } = render(
+      <MemoryRouter>
+        <App />
+      </MemoryRouter>
+    );
+    const gameNav = getByText('Game');
+    fireEvent.click(gameNav);
     const winningStreak = await waitFor(() => getByRole('heading', {name: "Winning streak: 0"}));
-    debug();
     expect(winningStreak).toBeInTheDocument();
 
-    //Still need to check if winning button (charmander) is clicked, should add 1 to win counter
+  
+    const winningBtn = await waitFor(() => getByRole('button', {name: "charmander"}));
+    fireEvent.click(winningBtn);
+    const winningStreak2 = await waitFor(() => getByRole('heading', { name: "Winning streak: 1" }));
```

Comment:
when you click winning button, winning streak is now 1

Reply:
Solid integration test. Could we take it a step further and check what happens when a streak ends?